en/docs/integration-guides/usecases/datamapper/read-csv-file-and-transform-to-xml-file.md

```diff
@@ -286,12 +293,13 @@ Click **Call Function** to insert this node into the automation flow.
 
 ## Step 7: Use `fileReadCsv` to Load the CSV File into Memory
```

Comment:
In the above `Click Call Function to insert this node into the automation flow.` , should start with `4. Click Call Function to insert this node into the automation flow.`